backend/apid/graphql/namespace.go

```diff
@@ -103,6 +103,48 @@ func (r *namespaceImpl) Checks(p schema.NamespaceChecksFieldResolverParams) (int
 	return res, nil
 }
 
+// Handlers implements response to request for 'handlers' field.
+func (r *namespaceImpl) Handlers(p schema.NamespaceHandlersFieldResolverParams) (interface{}, error) {
+	res := newOffsetContainer(p.Args.Offset, p.Args.Limit)
+	nsp := p.Source.(*types.Namespace)
+
+	// finds all records
+	filter := p.Args.Filter
+	results, err := loadHandlers(p.Context, nsp.Name)
+	if err != nil {
+		return res, err
+	}
+
+	records := filterHandlers(results, func(obj *types.Handler) bool {
+		if filter == "" {
+			return true
+		}
+		sobj := dynamic.Synthesize(obj)
+		matched, err := js.Evaluate(filter, sobj, nil)
+		if err != nil {
+			logger.WithError(err).Debug("unable to filter record")
+		}
+		if matched {
+			return true
+		}
+		return false
+	})
+	if err != nil {
+		return res, err
+	}
+
+	sort.Sort(types.SortHandlersByName(
+		records,
+		p.Args.OrderBy == schema.HandlerListOrders.NAME,
+	))
+
+	// paginate
+	l, h := clampSlice(p.Args.Offset, p.Args.Offset+p.Args.Limit, len(records))
+	res.Nodes = records[l:h]
+	res.PageInfo.totalCount = len(records)
+	return res, nil
+}
```

Comment:
I know this is a bit of 'do as I say, not as I do' but would you mind adding a test for this field resolver? There should be a test for the silences field resolver that you could use as a template.